REBUILD_GENERATOR_BRIEFING.md

```
@@ -0,0 +1,247 @@
+/Users/jamesfoote/Documents/PTP/Construction/mortgage-study-rag/.venv/bin/python -c "
+import openpyxl
+wb = openpyxl.load_workbook('PTP-Project-Setup-Template.xlsx')
+for i, name in enumerate(wb.sheetnames):
+    ws = wb[name]
+    print(f'{i}. {name!r}  (rows={ws.max_row}, cols={ws.max_column})')
+"
+```
+
+Expected output (this is the contract you're matching):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)